docs/changelog.md

```diff
@@ -5,6 +5,8 @@
 
   * `repo` and `mount` keys in app specs are now optional
 
+  * `always` and `once` scripts in app specs have their output logged to `/var/log/dusty_always.log` and `/var/log/dusty_once.log`, respectively. These are in-container locations
+
```

Comment:
Are you saying it needs to start with "NEW: "?  What is the above bullet point doing?